inference-engine/tests/functional/plugin/cpu/single_layer_tests/concat.cpp

```diff
@@ -2,68 +2,89 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "shared_test_classes/base/ov_subgraph.hpp"
 #include "ngraph_functions/builders.hpp"
 #include "test_utils/cpu_test_utils.hpp"
 
-using namespace InferenceEngine;
+using namespace ov::test;
 using namespace CPUTestUtils;
 
 namespace CPULayerTestsDefinitions {
 
+using ConcatInputShapes = std::pair<std::vector<ov::PartialShape>, std::vector<std::vector<ov::Shape>>>;
```

Comment:
Could you please rewrite the tests considering approach to the input shapes description that we discussed offline?

Reply:
from my point of view for tests with several inputs better to use this approach because it look more understandable in instantiate
**current**
{
            // dynamic
            // *0 input, 1 input, 2 input*
            {{-1, -1, -1, -1}, {-1, -1, -1, -1}, {-1, -1, -1, -1}},
            // target
            {
                // *0 input, 1 input, 2 input*
                {{2, 32, 5, 7}, {2, 16, 5, 7}, {2, 64, 5, 7}}, <======== 1 inference
                {{1, 18, 10, 2}, {1, 5, 10, 2}, {1, 45, 10, 2}},  <======== 2 inference
                {{3, 8, 1, 8}, {3, 3, 1, 8}, {3, 1, 1, 8}},  <======== 3 inference
            }
        },
        
**with InputShapes from base class**
*dynamic* *1 inference* *2 inference*
{{-1, -1}, {{55, 12}, {33, 7}}}, // *input 0*
{{-1, -1}, {{12, 55}, {7, 33}}}  // *input 1*

Reply:
May be, but we obviously better align our instances with common tests.

Reply:
got it, fixed